Since we don't have a "swig" python package to look for, this breaks the
build. Swig detection, however, is done locally by the ports system and
the configure script.

--- pyproject.toml.orig	2025-06-17 08:21:23 UTC
+++ pyproject.toml
@@ -59,5 +59,5 @@ content-type = "text/plain"
 
 [build-system]
 # Minimum requirements for the build system to execute.
-requires = ["setuptools", "swig"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
